publisher/image/Dockerfile

```diff
@@ -1,8 +1,4 @@
-FROM alpine:3.1
-
-# install curl in the image so it is possible to get the runtime
-# profiling information without any additional package installation.
-RUN apk add --update-cache curl && rm -rf /var/cache/apk/*
```

Comment:
@aledbf  Does it make sense to keep this in the image just for profiling? Is this something that could be exposed another way? Logs?